### PR TITLE
feat/checkbox

### PR DESCRIPTION
**Pull request by tassioFront**
## Infos

[Task](https://juntossomosmais.monday.com/boards/359803983/pulses/736106666)

#### What is being delivered?

Update Checkbox styles and create Checkbox component to Vue.

I've talk with Lucas Ramalho about new styles. He and Raquel Mendes have decided to not use error message, by them, it is unnecessary.

 The new style:
Captura de Tela 2020-09-22 às 11 29 50

#### What impacts?

Now, We could use Checkbox component at Vue projects.

#### Reversal plan

Revert it, make changes and update.

#### Evidences

Vue: 
Captura de Tela 2020-09-22 às 18 32 02

React:
Captura de Tela 2020-09-22 às 19 10 30

Error message:
Captura de Tela 2020-09-23 às 11 36 19

## DoD checklist

- [x] Related code finished
- [x] Documentation updated (if exists)
- [x] Unit tests written and passing
- [x] Any configuration or build changes documented
- [x] Project builds without errors
- [x] Lint errors fixed

### CODE REVIEW

**Review by elizeumatheus — APPROVED** [review]

**Comment by elizeumatheus** [comment]
I didn't find this in the files, but on react docs the component name is Input as showed on yours evidences hahahhaha maybe the evidence is just out of date, but 🤷🏾‍♂️ hahhahaha

**Comment by tassioFront** [comment]
> I didn't find this in the files, but on react docs the component name is Input as showed on yours evidences hahahhaha maybe the evidence is just out of date, but 🤷🏾‍♂️ hahhahaha

You're totally right! hahahahahaha I did not notice.
 It is in PROD too, I will remove.

**Comment by carloshrabelo** [comment]
Why you removed the error place?

File: packages/react-ds/src/components/Checkbox/index.tsx [hunk 1]
```diff
@@ -20,29 +18,21 @@ const InternalCheckbox = (
...
-            {label}
-          </label>
-        )}
-      </div>
-      {/* TODO: Replace for alert component */}
-      {error && (
-        <div className={styles.error}>
-          <Alert />
-          {error}
-        </div>
```
Comment (viniciuscr) [inline packages/react-ds/src/components/Checkbox/index.tsx:44]: question: if the error message is not showing here anymore, where it will be?

**Comment by tassioFront** [comment]
@carloshrabelo and @viniciuscr, Lucas Ramalho and Raquel Mendes have talked about it (error message) and he ask me to remove because, by them, we won't use. hehe 😜  I understood that we have not been use it, but if there is some place with it send me a print and I will show for them.

**Comment by viniciuscr** [comment]
> @carloshrabelo and @viniciuscr, Lucas Ramalho and Raquel Mendes have talked about it (error message) and he ask me to remove because, by them, we won't use. hehe 😜 I understood that we have not been use it, but if there is some place with it send me a print and I will show for them.

image

**Review by elizeumatheus — APPROVED** [review]

**Review by viniciuscr — DISMISSED** [review]

File: packages/styles/components/CheckboxRadio.module.scss [hunk 2]
```diff
@@ -1,26 +1,89 @@
...
+}
+
+.checkbox {
+  cursor: pointer;
+  height: 0;
+  opacity: 0;
+  width: 0;
+
+  &:disabled {
+    ~ .customCheckbox {
```
Comment (felipefialho) [inline packages/styles/components/CheckboxRadio.module.scss:31]: 😍

File: packages/styles/components/CheckboxRadio.module.scss [hunk 3]
```diff
@@ -1,26 +1,89 @@
 @import '~@venice/styles/core';
 
+$checkbox-border: 2px;
+$checkbox-size: 14px;
+$checkbox-opacity: 0.55;
```
Comment (felipefialho) [inline packages/styles/components/CheckboxRadio.module.scss:5]: Perfect!

**Review by felipefialho — DISMISSED** [review]
Don't forgot to test a generated build inside projects before merge it 😁

**Comment by tassioFront** [comment]
I need to wait add venice-icons at WebCustomer(LV) and Colosseum.

**Review by lupy100 — DISMISSED** [review]

**Review by viniciuscr — APPROVED** [review]

**Review by felipefialho — DISMISSED** [review]

**Comment by tassioFront** [comment]
> It needs to wait add venice-icons at WebCustomer(LV) and Colosseum.

**Review by elizeumatheus — DISMISSED** [review]

**Review by HelenDias — APPROVED** [review]

**Review by lupy100 — DISMISSED** [review]

**Review by elizeumatheus — DISMISSED** [review]

**Review by felipefialho — APPROVED** [review]

**Review by lupy100 — APPROVED** [review]

**Pull request merged**